### PR TITLE
Added static_cast to suppress compiler warning.

### PR DESCRIPTION
**Pull request by MitchBradley**
PR #939 compiles but there is a compiler warning about int being converted to unsigned int.  This PR makes that warning go away.

### CODE REVIEW

**Review by dymk — APPROVED** [review]
Yep, looks reasonable to me. The length should never overflow a size_t, nor could the value be negative.

File: FluidNC/src/Configuration/Tokenizer.h [hunk 1]
```diff
@@ -11,64 +11,36 @@
...
-        bool IsIdentifierChar() { return IsAlpha() || IsDigit() || Current() == '_'; }
-
-        bool IsEndLine() { return Eof() || Current() == '\n'; }
-
-        bool IsDigit() {
-            char c = Current();
-            return (c >= '0' && c <= '9');
-        }
+        bool isWhiteSpace(char c) { return c == ' ' || c == '\t' || c == '\f' || c == '\r'; }
+        bool isIdentifierChar(char c) { return (c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z') || (c >= '0' && c <= '9') || c == '_'; }
```
Comment (dymk) [inline FluidNC/src/Configuration/Tokenizer.h]: nit - move implementation into Tokenizer.cpp

**Review by dymk — APPROVED** [review]
Looks good to me, although I'm not super familiar with the ins and outs of FluidNC's yaml subset. Looks like the logical changes are confined to Tokenizer - nice to see some simplifications!

File: FluidNC/src/Configuration/Tokenizer.h (hunk 1 above)
Comment (MitchBradley) [reply FluidNC/src/Configuration/Tokenizer.h]: Done.

**Comment by MitchBradley** [comment]
> I'm not super familiar with the ins and outs of FluidNC's yaml subset

In a nutshell - the only value data type at the tokenizing level is string.  Values are converted from string to another type on a per-key basis.  There are no arrays.  So it's a very restricted subset.

**Pull request merged**